validator/testdata/transformed_feature_tests/i_amphtml_sizer_child.html

```diff
@@ -56,5 +56,18 @@
      <amp-img src="/img/canoe_900x600_blur.jpg" alt="A blurry image about canoeing" layout="fill" class="i-amphtml-layout-fill i-amphtml-layout-size-defined" i-amphtml-layout="fill"></amp-img>
      <amp-img src="/img/canoe_900x600.jpg" alt="An image about canoeing" layout="fill" class="i-amphtml-layout-fill i-amphtml-layout-size-defined" i-amphtml-layout="fill"></amp-img>
    </amp-image-slider>
+
+  <!-- Valid: amp-image-slider > i-amphtml-sizer[slot="i-amphtml-svc"] -->
+  <amp-image-slider width="300" height="200" layout="responsive" class="i-amphtml-layout-responsive i-amphtml-layout-size-defined" i-amphtml-layout="responsive">
+    <i-amphtml-sizer slot="i-amphtml-svc" style="display:block;padding-top:66.6667%;"></i-amphtml-sizer>
+    <amp-img src="/img/canoe_900x600_blur.jpg" alt="A blurry image about canoeing" layout="fill" class="i-amphtml-layout-fill i-amphtml-layout-size-defined" i-amphtml-layout="fill"></amp-img>
+    <amp-img src="/img/canoe_900x600.jpg" alt="An image about canoeing" layout="fill" class="i-amphtml-layout-fill i-amphtml-layout-size-defined" i-amphtml-layout="fill"></amp-img>
+  </amp-image-slider>
+
+  <!-- Valid: amp-video-iframe > i-amphtml-sizer[slot="i-amphtml-svc"] -->
+  <amp-video-iframe id="myVideo" src="/amp-video-iframe-videojs.html" width="720" height="405" layout="responsive" autoplay="" class="i-amphtml-layout-responsive i-amphtml-layout-size-defined" i-amphtml-layout="responsive">
+    <i-amphtml-sizer slot="i-amphtml-svc" style="display:block;padding-top:56.2500%;"></i-amphtml-sizer>
+    <amp-img placeholder="" src="/img/amp-video-iframe-sample-placeholder.jpg" layout="fill" class="i-amphtml-layout-fill i-amphtml-layout-size-defined" i-amphtml-layout="fill"></amp-img>
+  </amp-video-iframe>
```

Comment:
Should a couple invalid examples be added here as well? Or is that excessive?

Reply:
I would not consider it excessive and would encourage it.

Reply:
Encouragement received! Added invalid test cases.